src/sentry/notifications/utils/participants.py

```diff
@@ -430,15 +462,15 @@ def get_users_from_team_fall_back(
     user_ids: set[int] = set()
     for team in teams_to_fall_back:
         # Fall back to notifying each subscribed user if there aren't team notification settings
-        member_list = team.member_set.values_list("user_id", flat=True)
-        user_ids |= set(member_list)
+        members = organization_service.get_team_members(team_id=team.id)
```

Comment:
This new call to `organization_service.get_team_members` is the only change I had to make to compensate for the loss of access to the original `Team` object. But considering that the old code was making a database query here before, it is probably a wash on cost.